CRM/Civisocial/OAuthProvider/Facebook.php

```diff
@@ -160,11 +167,8 @@ public function handleCallback() {
         'created_date' => time(), // @todo: Created Date not being recorded
       );
 
-      CRM_Civisocial_BAO_CivisocialUser::create($socialUser);
+      civicrm_api3('CivisocialUser', 'create', $socialUser);
     }
-
```

Comment:
i would add this code in parent class since this code snippet is same for all 3 network types